user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md

```diff
@@ -4,52 +4,59 @@ uid: BPA_SRM_Framework_Configuration
+
 ### Error
 
-* "Booking doesn't have Property with names: 'Booking Manager', 'Booking Life Cycle', 'FriendlyReference', 'PreRoll', 'PostRoll', 'Service State', 'Start', 'End', 'VisualBackground' and 'VisualForeground'"
-* "Booking has an empty value for Property 'Booking Manager', 'Booking Life Cycle', 'FriendlyReference', 'PreRoll', 'PostRoll', 'Service State', 'Start', 'End', 'VisualBackground' and 'VisualForeground'"
-* "Properties 'Start' and 'End' are not in DateTime format"
-* "Properties 'PreRoll' and 'PostRoll' are not in TimeSpan format"
-* "'Start' - 'PreRoll' Property values is different from Reservation.Start"
-* "'End' + 'PostRoll' Property values is different from Reservation.End when 'End' Property different from 'Not Applicable'"
-* "'PostRoll' Property value is different from TimeSpan.Zero when 'End' Property is equal to 'Not Applicable'"
-* "'Booking Manager' Property value refers to an existing DataMiner element using connector 'Skyline Booking Manager' and that element is not Active"
-* "Booking doesn't have 'Virtual Platform' Property or this has a value different from the Element 'Default Virtual Platform' parameter"
+One or more issues have been found:
+
+- The booking does not have properties with the name *Booking Manager*, *Booking Life Cycle*, *FriendlyReference*, *PreRoll*, *PostRoll*, *Service State*, *Start*, *End*, *VisualBackground*, and/or *VisualForeground*.
+- The booking has an empty value for the properties *Booking Manager*, *Booking Life Cycle*, *FriendlyReference*, *PreRoll*, *PostRoll*, *Service State*, *Start*, *End*, *VisualBackground*, and/or *VisualForeground*".
+- The properties *Start* and *End* are not in DateTime format.
+- The properties *PreRoll* and *PostRoll* are not in TimeSpan format.
+- The *Start* - *PreRoll* property value is different from *Reservation.Start*.
+- The *End* + *PostRoll* property value is different from *Reservation.End* while the *End* property does not have the value *Not Applicable*.
+- The *PostRoll* property value is different from TimeSpan.Zero while the *End* property has the value *Not Applicable*.
+- The *Booking Manager* property value refers to an existing DataMiner element using the connector *Skyline Booking Manager* but that element is not active.
+- The booking does not have the *Virtual Platform* property or does have this property but with a value that is different from the element's *Default Virtual Platform* parameter.
 
 ### Warning
 
 This BPA does not generate warnings.
 
 ### Not Executed
 
-* "Resource Manager is not initialized"
+If the test fails to execute, this message is displayed:
+
+- `Resource Manager is not initialized.`
 
 ## Impact when issues detected
 
-- Impact: "Bookings may not execute correctly."
-- Action: "Please contact 'support.automation-orchestration@skyline.be' for support."
+- Impact: Bookings may not execute correctly.
+- Action: Please contact <support.automation-orchestration@skyline.be> for support.
```

Comment:
Should be 'Please contact your system administrator for support.', forgot to change that here, my mistake